src/authenticated-app.js

```diff
@@ -36,8 +37,13 @@ function ErrorFallback({ error }) {
 function AuthenticatedApp() {
   const { user } = useSelector(selectUser);
   const dispatch = useDispatch();
-  const handleLogout = () => dispatch(logout());
-  React.useEffect(() => dispatch(getBookListByQuery('')), [dispatch]);
+  const handleLogout = () => {
+    dispatch(logout());
+  };
+  React.useEffect(() => {
+    dispatch(getBookListByQuery(''));
+    dispatch(getListItem());
+  }, [dispatch]);
```

Comment:
It's better if we create 2 reset actions for the book and list items. If we fetch after the user logged in, the data is still cached before the API return response.

Reply:
Or we can handle `logout` action in `extraReducers`.